views/components/HeroFooter/HeroFooter.css

```diff
@@ -0,0 +1,13 @@
+.footer_container {
```

Comment:
the block for `Hero` is `HeroCard`, the block for `HeroFooter` is `footer`. Consistency is important. Let's always make the block the same name and casing of the component name. In this case, `HeroFooter`.

Reply:
[9d7c4dc](https://github.com/AMP-SCZ/dpdash/pull/468/commits/9d7c4dc05024fa07956aaa578a84f844f590db07)